### PR TITLE
Contributing.md

### PR DESCRIPTION
**Pull request by fleupold**
# Description
We didn't yet have a CONTRIBUTING.md in this repo and it lends itself to specify the rewards for "help-wanted" tasks.

The text is mainly borrowed from https://github.com/cowprotocol/repo-template/blob/main/CONTRIBUTING.md with minor additions

# Changes
- [x] Add CONTRIBUTING.md

## How to test
Read the file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+When opening the pull request you will be presented with a template and a series of instructions. Read through it carefully and follow all the steps. 
+Expect a review and feedback from the maintainers afterwards.
+
+If you're looking for a good place to start, look for issues labelled ["help wanted"](./labels/help%20wanted)!
+
+## Reward for Contributions
+We are excited to support our community's contributions in a tangible way. 
+For merged pull requests that contribute to issues labeled "help wanted", we are offering a reward of 100 DAI.
+To be eligible for this reward, make sure your pull request meets all the required criteria and is successfully merged.
```
Comment (MartinquaXD) [inline CONTRIBUTING.md:39]: If I were a contributor reading this I would like to have a little bit more information on the process.
Primarily on which chain and by which way we reach out the user.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+You can [open an issue] to suggest a feature or report a minor bug.
+
+If you believe your issue may be due to user error and not a problem in the library, consider instead posting a question on the [CoW Protocol Discord].
+
+Before opening an issue, be sure to search through the existing open and closed issues, and consider posting a comment in one of those instead.
+
+When requesting a new feature, include as many details as you can, especially around the use cases that motivate it. 
+Features are prioritised according to the impact they may have on the ecosystem, so we appreciate information showing that the impact could be high.
+
+[open an issue]: ./issues/new/choose
```
Comment (MartinquaXD) [inline CONTRIBUTING.md:23]: This link and others doesn't work probably due to the relative path. I assume they'd work after merging?

**Review by MartinquaXD — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Any non-trivial documentation must be first discussed with the maintainers in an issue (see [Opening an issue](#opening-an-issue)). Only very minor changes are accepted without prior discussion.
+
+When opening the pull request you will be presented with a template and a series of instructions. Read through it carefully and follow all the steps. 
+Expect a review and feedback from the maintainers afterwards.
+
+If you're looking for a good place to start, look for issues labelled ["help wanted"](https://github.com/cowprotocol/services/labels/help%20wanted)!
+
+## Reward for Contributions
+We are excited to support our community's contributions in a tangible way. 
+For merged pull requests that contribute to issues labeled "help wanted", we are offering a reward of 100 DAI.
```
Comment (squadgazzz) [inline CONTRIBUTING.md:39]: That would be helpful to specify the exact discord channel, forum thread or any other way to claim the reward.

**Review by squadgazzz — APPROVED** [review]

**Review by sunce86 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (fleupold) [reply CONTRIBUTING.md:39]: It says below to leave the Gnosis Chain address either on the PR or for the more privacy concerned people message it to mastercow.eth

**Pull request merged**